### ISSUE TITLE
🐛Bug: radio group accessible label has problems

### ISSUE THREAD

**Issue by Wolfr** [melt-ui/melt-ui#301]
**Describe the bug**
Aria-label used in a wrong manner in radio group

**Expected behavior**
A screen reader will recognize the "View density" radio group and recognize the underlying fields 

**Additional context**
I believe the code as it is on https://www.melt-ui.com/docs/builders/radio-group will cause a screen reader to read the whole section as "Screen density" and make the underlying radio buttons unreachable. I.e. it collapses the whole section to just be the words "screen density". 

For more info see

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Specifically 

_Not all elements can be given an accessible name. Neither aria-label nor aria-labelledby should be used with non-interactive elements or inline structural role such as with code, term, or emphasis nor roles whose semantics will not be mapped to the accessibility API, including presentation, none, and hidden. The aria-label attribute is intended for interactive elements only._

I can add a test video when I am on desktop

**Comment by Wolfr** [comment]
Nevermind, I don't think it's a problem because there is the associated aria-role of radio group.
